pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -260,26 +257,18 @@ func storageProvisionerKey(sc *storagev1.StorageClass) string {
 }
 
 var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageClass) string{
-	"pxd.openstorage.org": func(sc *storagev1.StorageClass) string {
-		// https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs/
-		val := sc.Parameters["shared"]
-		if val == "true" {
-			return "pxd.openstorage.org/shared"
-		}
-		return "pxd.openstorage.org"
-	},
 	"kubernetes.io/portworx-volume": func(sc *storagev1.StorageClass) string {
-		val := sc.Parameters["shared"]
-		if val == "true" {
-			return "kubernetes.io/portworx-volume/shared"
+		opts := strings.Split(sc.Parameters["sharedv4_mount_options"], ",")
+		if slices.Contains(opts, "vers=3.0") && slices.Contains(opts, "nolock") {
```

Comment:
is using `slices` going to be backportable? just saving you some effort here

Reply:
It's in since go 1.18, so I guess we should better it, and if needed for too-old-backports we'll implement it there.

Reply:
It's in since 1.21 https://stackoverflow.com/a/71181131

Reply:
Right, in 1.18 it was still experimental (`import  "golang.org/x/exp/slices"`).